helm_deploy/whereabouts-api/templates/deployment.yaml

```diff
@@ -25,6 +26,15 @@ spec:
         app: {{ template "app.name" . }}
         release: {{ .Release.Name }}
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - {{ template "app.name" . }}
```

Comment:
Note that we don't need to specify the namespace since from https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity:
> If omitted or empty, it defaults to the namespace of the pod where the affinity/anti-affinity definition appears.
which means that it will only try to arrange pods in the same namespace on different nodes